<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTextEdit" name="chatField">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>341</width>
        <height>511</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>550</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>550</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>clients</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>your chat rooms</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>221</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>private</string>
       </attribute>
       <widget class="QListWidget" name="privateRoomsList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>201</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>public</string>
       </attribute>
       <widget class="QListWidget" name="publicRoomsList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>201</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>201</width>
        <height>221</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>global</string>
       </attribute>
       <widget class="QListWidget" name="registeredUsersList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>201</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>room</string>
       </attribute>
       <widget class="QListWidget" name="roomMembersList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>201</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="startChattingBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>start chatting with user</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="createPublicRoomBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>create public room</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addUserToRoomBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>add user to public room</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="loginLineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>login </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signinBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>241</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Don't have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gotosignupBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>sign up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signupBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>create account</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>login </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginLineEdit_signup">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit_signup">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit_signup">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>repeat password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="repeatpasswordLineEdit_signup">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Already have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gotosigninBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLineEdit" name="serverHostLE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverPortLE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>301</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>7999</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>31</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>host</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>31</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>connect to server</string>
      </property>
     </widget>
     <widget class="QPushButton" name="defaultServerBtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>281</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>use default remote server*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectBtn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>211</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>21</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>OR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>*won't work</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
